.vscode/launch.json

```diff
@@ -19,6 +19,28 @@
                 "--dut-definitions=../dut-definitions/"
             ]
         },
+        {
```

Comment:
Please remove this change. It's very specific to your environment (devkit) and shouldn't be in the repo. I feel the same about the ones we have already which have snuck but that's for another PR ;)
I have not yet found a solution for this unfortunately.

Reply:
Totally agree :)